### PR TITLE
Done with team section and how it works section

### LINKED ISSUE #46: team section
* Reorganize team section to have picture on the left and team pictures on the right

### LINKED ISSUE #50: shifting each section space out to link to header nicely

### PR DESCRIPTION
**Pull request by Ashley-Lau**
- Sean needs to insert his picture, and Keane can change or resize his picture himself to make the picture clearer.
- Need to insert logo/ images for each step in how it works section.

### CODE REVIEW

**Comment by seanjyjy** [comment]
* Perhaps for Team and How it work section, they can be more centralised, or the text feels too to the right
* For Team, leave more space between Header and content
* Images for team members feels weird

Overall Good Job

File: src/howitworks/HowItWorks.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  return (
+    <div id="howItWorks" className="how-it-works">
+      <HowItWorksHeader sectionHeader={'How it Works'} translateX={'translateX(28px)'} />
+      <div className="how-it-works-container">
+        <img className="how-it-works-picture" src={HowItWorksImage} />
+        <div className="steps">
+          <Step
+            // img={}
+            title="1. Select the Algorithm"
+            description="Visualise the algorithm of your choice."
```
Comment (keanecjy) [inline src/howitworks/HowItWorks.js]: I believe we should use visualize instead to keep consistent with 'Visualizer'

File: src/howitworks/HowItWorks.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          <Step
+            // img={}
+            title="3. Play"
+            description="Press the play button to start the step by step sorting process."
+          />
+        </div>
+        <div className="steps">
+          <Step
+            // img={}
+            title="2. Select Defaults"
```
Comment (keanecjy) [inline src/howitworks/HowItWorks.js]: I think `Defaults` may not be quite right? Perhaps it should be configuration / settings?

File: src/howitworks/HowItWorks.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            description="Adjust the speed of the visualizer and array size."
+          />
+          <Step
+            // img={}
+            title="4. Fast-forward, Backtrack, and Replay"
+            description="Replay the entire sorting process or go walk through
+              the algorithm on step at a time with these functions."
```
Comment (keanecjy) [inline src/howitworks/HowItWorks.js]: Some typos here. 
Should be:
* Replay the entire sorting process or ~~go~~ walk through
              the algorithm ~~on~~ one step at a time with these functions.

**Review by keanecjy — CHANGES_REQUESTED** [review]
Great work overall. Agree with the points mentioned by @seanjyjy 

Also would like to add that we can perhaps have more space between each section (Home and How it Works, Visualizer and Team)

**Comment by Ashley-Lau** [comment]
Resolved the typos and improved on styling

**Review by keanecjy — APPROVED** [review]
LGTM!

**Pull request merged**